AppHandling/Run-AlPipeline.ps1

```diff
@@ -174,6 +176,31 @@
   Override function parameter for Import-TestToolkitToBcContainer
  .Parameter CompileAppInBcContainer
   Override function parameter for Compile-AppInBcContainer
+ .Parameter PreCompileApp
+  Custom script to run before compiling an app.
+  The script should accept the type of the app and a reference to the compilation parameters.
+  Possible values for $appType are: app, testApp, bcptApp
+  Example:
+  {
+    param(
+        [string] $appType,
+        [ref] $compilationParams
+    )
+    ...

```

Comment:
Add an example on how to use compilationParams here:

$path = $compilationParams.Value.appProjectFolder

Reply:
Isn't that more like how to use `ref`? As in [about_Ref](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ref?view=powershell-5.1).

I thought of adding some doc around what parameters are available (e.g. `appProjectFolder`, `appOutputFolder` etc.) these were never documented anyway.

Reply:
Added an example in the parameter's doc.